shell/platform/darwin/ios/framework/Source/FlutterOverlayView.mm

```diff
@@ -50,18 +47,10 @@ - (instancetype)init {
 - (instancetype)initWithContentsScale:(CGFloat)contentsScale {
   self = [self init];
 
-  if ([self.layer isKindOfClass:[CAEAGLLayer class]]) {
-    CAEAGLLayer* layer = reinterpret_cast<CAEAGLLayer*>(self.layer);
-    layer.allowsGroupOpacity = NO;
```

Comment:
Why are we dropping this here?  According to https://developer.apple.com/documentation/quartzcore/calayer/1621277-allowsgroupopacity?language=objc this defaults to `YES` on iOS 7.0+.